index.html

```diff
@@ -85,13 +85,23 @@ <h3>Party</h3>
             <a href="" class="all-program">SEE THE WHOLE PROGRAM</a>
```

Comment:
- [ ] Well done on implementing the "SEE THE WHOLE PROGRAM" button for the desktop version. However, as mentioned by the previous reviewer, it's evident that the button design for the mobile version differs significantly. To ensure consistency and adherence to the design, I recommend finding a solution to display the button as specified in the design for the mobile version. You have a couple of options to achieve this: you can dynamically create the button using JavaScript based on screen size, or you can apply CSS styling to achieve the desired layout. Choose the approach that aligns best with your project's structure and requirements. By addressing this difference, you'll ensure a cohesive and polished user interface. Keep up the great work! 🌟📝🚀

| Your | Design |
| --- | --- |
|  ![image](https://github.com/amiralda/haitian-Otaku-Website/assets/55840999/a5a806e3-8f12-43d8-a1ba-a04bb67401cf)| ![image](https://github.com/amiralda/haitian-Otaku-Website/assets/55840999/60d4644d-a7b9-421c-b2c3-b2ce522dd99a)|

Reply:
- [ ] Nice work changing the button style on the mobile version :clap: However your button design looks different from the one in the provided design. And Kindly note that the button should only show on the mobile version and a simple link is shown on the desktop version. Therefore I suggest you style the button to look exactly like the one on the provided design.
### Your design :point_down: 
![Screenshot from 2023-08-11 07-16-26](https://github.com/amiralda/haitian-Otaku-Website/assets/91142432/204e141c-3716-40db-babc-46e3e9e0e37d)
### What it is supposed to look like :point_down: 
![](https://user-images.githubusercontent.com/55840999/259918849-60d4644d-a7b9-421c-b2c3-b2ce522dd99a.png)